### PR TITLE
Fix failing httplistener tests on Win7.

### LINKED ISSUE #18192: HttpListener tests: HttpRequestStreamTests failed with "System.NullReferenceException"
Opened on behalf of @Jiayili1

The test `System.Net.Tests.HttpRequestStreamTests/CanRead_Get_ReturnsTrue` has failed.

System.AggregateException : One or more errors occurred. (Object reference not set to an instance of an object.) (Object reference not set to an instance of an object.)\r
    ---- System.NullReferenceException : Object reference not set to an instance of an object.\r
    ---- System.NullReferenceException : Object reference not set to an instance of an object.

        Stack Trace:

        
    ----- Inner Stack Trace #1 (System.NullReferenceException) -----
       at System.Net.Tests.HttpRequestStreamTests.<CanRead_Get_ReturnsTrue>d__13.MoveNext()
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    --- End of stack trace from previous location where exception was thrown ---
       at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
       at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
       at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
    ----- Inner Stack Trace #2 (System.NullReferenceException) -----
       at System.Net.Tests.HttpRequestStreamTests.Dispose()
       at ReflectionAbstractionExtensions.DisposeTestClass(ITest test, Object testClass, IMessageBus messageBus, ExecutionTimer timer, CancellationTokenSource cancellationTokenSource)
Build : Master - 20170411.01 (Core Tests)
Failing configurations:
- Windows.7.Amd64
  - x86-Debug
  - x64-Debug

Detail: https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20170411.01/workItem/System.Net.HttpListener.Tests/analysis/xunit/System.Net.Tests.HttpRequestStreamTests~2FCanRead_Get_ReturnsTrue

### PR DESCRIPTION
**Pull request by Priya91**
fixes #18192 

cc @hughbe

### CODE REVIEW

**Comment by Priya91** [comment]
@dotnet-bot test outerloop netcoreapp Windows 7 Debug
@dotnet-bot test outerloop netcoreapp Ubuntu14.04 Debug
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

File: src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs [hunk 1]
```diff
@@ -478,4 +475,3 @@ public void Dispose()
         }
     }
 }
-
```
Comment (davidsh) [inline src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs]: nit: newline missing at end of file

**Comment by davidsh** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug

File: src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs (hunk 1 above)
Comment (Priya91) [inline src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs]: It should be fixed in my change on the right?

**Pull request merged**